doc: Update versions for 2.2.0 release (#877)
nrfconnect/sdk-nrfxlib: lc3/CHANGELOG.rst

> All notable changes to this project are documented in this section.

Reviewer: ```suggestion
All notable changes to this project are documented on this page.
```

nRF Connect SDK v2.2.0
**********************

> All the notable changes included in the |NCS| v2.2.0 release are documented in this section.

Reviewer: ```suggestion
Here are all the notable changes included in the |NCS| v2.2.0 release:
```
